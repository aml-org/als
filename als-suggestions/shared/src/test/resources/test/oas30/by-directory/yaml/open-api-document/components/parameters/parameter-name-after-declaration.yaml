openapi: 3.0.0
components:
  parameters:
      test:
        description: a parameter
      *