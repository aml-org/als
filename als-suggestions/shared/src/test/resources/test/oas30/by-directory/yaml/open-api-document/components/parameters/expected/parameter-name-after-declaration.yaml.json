[
  {
    "label": "New parameter",
    "kind": [
      
    ],
    "detail": "template",
    "documentation": "",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "20New parameter",
    "filterText": "$1:\n  required: $2\n  name: $3\n  description: $4\n  in: $5",
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 5,
          "character": 6
        },
        "end": {
          "line": 5,
          "character": 6
        }
      },
      "newText": "$1:\n  required: $2\n  name: $3\n  description: $4\n  in: $5"
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]